packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -830,6 +830,9 @@ public void setIndoorEnabled(boolean indoorEnabled) {
 
   public void setTrafficEnabled(boolean trafficEnabled) {
     this.trafficEnabled = trafficEnabled;
+    if (googleMap != null) {
+      googleMap.setTrafficEnabled(trafficEnabled);
+    }
```

Comment:
Nits: We prefer escape early. e.g:
```java
if (googleMap == null) {
  return;
}
googleMap.setTrafficEnabled(trafficEnabled);
```
I'd appreciate it if you can update it as well :)